### DISCUSSION TITLE
4/22(토) Codable, Encoding, Decoding

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#97] [General]
요번 시험 끝나면 네트워크를 해야되니 조금이라도 맛봐 놓으면 좋을 것 같아서 Codable과 Encoding, Decoding을 냈습니다. 주로 Data Model을 구조체로 정의 할 때 사용하니까 확실하게 개념을 잡으면 명세서만 봐도 이걸로 하면 되겠네 라고 생각할 수 있으면 이해 잘 하시고 넘어가셨으면 합니다. 그리고 가능하면 예제 코드도 만들어 보는 것을 추천합니다 🎧

**Comment by Eunho0922** [comment]
넵
